src/components/Alerts/locationsCOVID_19.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { alert as i18n1 } from 'js/i18n/definitions';
+import ExternalLink from 'components/ExternalLink';
+
+const LocationsCOVID_19 = (x) => {
```

Comment:
x ?

Reply:
... Marks the spot! Yes, I'm sure that never made any sense. Gonna fix this now.